config/proto/targets.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
+
+package hermes.targets
+
+option go_package = "github.com/googleinterns/hermes/targets/proto";
+
+message TargetsDef {
+  enum TargetSystem {
+    GOOGLE_CLOUD_STORAGE = 0;
+    CEPH = 1;
+  }
+
+  enum ConnectionType {
+   HTTP = 0;
+   HTTPS = 1;
+  }
+
+  required TargetSystem target_system = 1;
+  ConnectionType connection_type = 2;
+  ConnectionDetails connection_details = 3;
+  required int64 total_space_allocated = 4;
+}
+
+message ConnectionDetails {
+  string target_url = 1;
+  int port = 2;
+  string api_key = 3;
```

Comment:
```suggestion
  required string target_url = 1;
  optional int port = 2; // Required for Ceph, but not for GCS since we will use the GCS go client library.
  optional string api_key = 3; // Required for Ceph, not for GCS which uses service account credentials in an environment variable.
```